TRITON-1665 MetricsManager should stop EventloopMetrics when it closes

melloc opened the pull request.

joshwilsdon reviewed (approved):
lgtm. Thanks for doing this.

melloc committed:
TRITON-1665 MetricsManager should stop EventloopMetrics when it closes
Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
Approved by: Josh Wilsdon <josh@wilsdon.ca>
